packages/app/Dockerfile

```diff
@@ -2,6 +2,10 @@ ARG NODE_VERSION=10.15.3
 
 FROM node:${NODE_VERSION} as builder
 
+
+ARG REACT_APP_API_URL=http://egapro-api:4000
+ENV REACT_APP_API_URL="${REACT_APP_API_URL}"
+
```

Comment:
This trick makes it possible to both have a default env var during the docker build phase (which builds the static files for the frontend, which will fetch the proxied `/api` instead of `http://localhost:4000), and an overloaded value when starting the image either with docker-compose or on k8s.